docs/design/w3c/w3c-representation.md

```diff
@@ -0,0 +1,457 @@
+## Design for W3C representation of AnonCreds credentials and presentation
+
+Currently `anoncreds-rs` library only provides support for Indy styled AnonCreds credentials matching to
+the [specification](https://hyperledger.github.io/anoncreds-spec/).
+
+This design document proposes extending of `anoncreds-rs` library to add support for AnonCreds W3C representation of
+verifiable credential and presentation described in the [document]().
+
+### Goals and ideas
+
+* Use Indy styled credentials to generate W3C AnonCreds credentials and presentations
+    * Credentials conversion:
+        * Convert Indy styled AnonCreds credentials into W3C form
+        * Convert W3C styled AnonCreds credentials into Indy form
+    * Presentation conversion (Optional):
+        * Convert Indy styled AnonCreds presentation into W3C form
+        * Convert W3C styled AnonCreds presentation into Indy form
+* Extend W3C credentials:
+    * Ability to set Data Integrity proof signatures for generated W3C credential objects:
+        * W3C credentials may contain multiple signatures
+        * AnonCreds-Rs only generates/handle AnonCreds signatures
+
+#### Out of scope
+
+* Credentials: Verify validity of the Data Integrity proof signature
+* Presentations: Create presentation using Data Integrity proof signature
+* Presentations: Verify validity of presentations using Data Integrity proof signatures
+* Presentations: Support different formats (for example DIF) of Presentation Request
+
+### Question impacting the approach
+
+* Q1: Do we need conversion for intermediate messages to make them W3C compliant: Credential Offer, Credential Request?
```

Comment:
In my opinion — no.  The only things that need to be W3C VC compliant are the VC and VP.